src/include/rollback_to_stable.h

```diff
@@ -109,6 +109,12 @@ struct __wt_rollback_to_stable {
     WT_THREAD_GROUP thread_group;
     uint32_t threads_num;
 
+    /*
+     * RTS threads_num configuration at connection level, by default the value is 0 representing not
```

Comment:
How about something like `The configuration of RTS worker threads at the connection level`? As per the above comment, we are going with some workers by default.